sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveExplainSuite.scala

```diff
@@ -101,4 +101,19 @@ class HiveExplainSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
         "Physical Plan should not contain Subquery since it's eliminated by optimizer")
     }
   }
+
+  test("SPARK-14251: EXPLAIN CODEGEN command") {
```

Comment:
Ok.

Regarding the name of the test. I would typically only add the JIRA number to test name if it fixes a bug (that needs some explanation).